BUG: series.iteritems deprecated

Error in viz_tests.py because iteritems has been deprecated from GeoSeries.

```
tests/viz_tests.py:26: in <module>
    ls_df = gpd.GeoDataFrame(geometry=utils.gaussian_linestrings(p_srs.geometry))
geoplot/utils.py:74: in gaussian_linestrings
    polys = [poly for _, poly in gaussian_polygons(points).iteritems()]
../../../miniconda3/envs/geoplot-dev/lib/python3.12/site-packages/pandas/core/generic.py:6299: in __getattr__
    return object.__getattribute__(self, name)
E   AttributeError: 'GeoSeries' object has no attribute 'iteritems'
========================================================================
```

--- a/geoplot/utils.py
+++ b/geoplot/utils.py
@@ -71,7 +71,7 @@ def gaussian_linestrings(points):
     Returns a GeoSeries of len(points) / 2 `shapely.geometry.LineString` objects for an array of
     `shapely.geometry.Point` objects.
     """
-    polys = [poly for _, poly in gaussian_polygons(points).iteritems()]
+    polys = [poly for _, poly in gaussian_polygons(points).items()]
     linestrings = []
     for poly in polys:
         start_point = shapely.geometry.Point(poly.exterior.coords[0])

